plugins/org.jkiss.dbeaver.ext.oracle/src/org/jkiss/dbeaver/ext/oracle/model/OracleChangeUserPasswordManager.java

```diff
@@ -41,7 +41,7 @@ public void changeUserPassword(@NotNull DBRProgressMonitor monitor, @NotNull Str
         // Do not use numbers in the password beginning
         try (JDBCSession session = DBUtils.openMetaSession(monitor, dataSource, "Change user password")) {
             session.enableLogging(false);
-            JDBCUtils.executeSQL(session, "ALTER USER " + DBUtils.getQuotedIdentifier(dataSource, userName) + " IDENTIFIED BY " + DBUtils.getQuotedIdentifier(dataSource, CommonUtils.notEmpty(newPassword)) +
+            JDBCUtils.executeSQL(session, "ALTER USER " + DBUtils.getUnQuotedIdentifier(dataSource, userName) + " IDENTIFIED BY " + DBUtils.getQuotedIdentifier(dataSource, CommonUtils.notEmpty(newPassword)) +
```

Comment:
What if the user name has characters that require the identifier to be quoted? I doubt it would work right in this case